src/pluto/image_rgba.cr

```diff
@@ -1,7 +1,4 @@
 require "./image"
-require "./bindings/*"
-require "./format/*"
-require "./operation/*"
```

Comment:
Removing duplicate `require`s, since `Image` already brings it formats and operations.